Node.js/sampleresponse.js

```diff
@@ -1,5 +1,5 @@
 /**
```

Comment:
Seems that the tx changes are inherited from boilerplate.js which doesn't seem right